socorro/external/statsd/statsd_rule_benchmark.py

```diff
@@ -2,50 +2,49 @@
 # License, v. 2.0. If a copy of the MPL was not distributed with this
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
-from socorro.external.statsd.statsd_base import (
-    StatsdBenchmarkingWrapper as StatsdRuleBenchmarkWrapper,
-)
```

Comment:
This was here solely for backwards compatibility of names. I nixed it and updated the rest of the code base accordingly.

I also verified we're not using this name in our configuration.